core/src/main/java/com/klaytn/caver/rpc/Klay.java

```diff
@@ -0,0 +1,1295 @@
+     * @param blockNumber The block number.
+     * @return KlayBlock
+     */
+    public Request<?, KlayBlock> getBlockByNumber(long blockNumber) {
+        return getBlockByNumber(blockNumber, true);
+    }
+
+    /**
+     * Returns information about a block by block number.
+     * It set "isFullTransaction" param to true.
+     * @param blockNumber The block number.
+     * @param isFullTransaction If true it returns the full transaction objects, if false only the hashes of the transactions.
+     * @return KlayBlock
+     */
+    public Request<?, KlayBlock> getBlockByNumber(long blockNumber, boolean isFullTransaction) {
+        DefaultBlockParameterNumber blockParameterNumber = new DefaultBlockParameterNumber(blockNumber);
+        return getBlockByNumber(blockParameterNumber, isFullTransaction);
+    }
+
+    /**
+     * Returns information about a block by block number.
+     * It set "isFullTransaction" param to true.
+     * @param blockTag The string "latest", "earliest" or "pending"
+     * @return KlayBlock
+     */
+    public Request<?, KlayBlock> getBlockByNumber(DefaultBlockParameter blockTag) {
+        return getBlockByNumber(blockTag, true);
+    }
+
+    /**
+     * Returns information about a block by block number.
+     * @param defaultBlockParameter The string "latest", "earliest" or "pending"
+     * @param isFullTransaction If true it returns the full transaction objects, if false only the hashes of the transactions.
+     * @return KlayBlock
+     */
+    public Request<?, KlayBlock> getBlockByNumber(DefaultBlockParameter defaultBlockParameter, boolean isFullTransaction) {
+        return new Request<>(
+                "klay_getBlockByNumber",
+                Arrays.asList(defaultBlockParameter, isFullTransaction),
+                web3jService,
+                KlayBlock.class);
+    }
+
+    /**
+     * Returns information about a block by block number.
+     * It set "isFullTransaction" param to true.
+     * @param blockHash The hash of block.
+     * @return KlayBlock
+     */
+    public Request<?, KlayBlock> getBlockByHash(String blockHash) {
+        return getBlockByHash(blockHash, true);
+    }
+
+    /**
+     * Returns information about a block by block number.
+     * @param blockHash The hash of block.
+     * @param isFullTransaction If true it returns the full transaction objects, if false only the hashes of the transactions.
+     * @return KlayBlock
+     */
+    public Request<?, KlayBlock> getBlockByHash(String blockHash, boolean isFullTransaction) {
+        return new Request<>(
+                "klay_getBlockByHash",
+                Arrays.asList(blockHash, isFullTransaction),
+                web3jService,
+                KlayBlock.class);
+    }
+
+    /**
+     * Returns receipts included in a block identified by block hash.
+     * @param blockHash The hash of block.
+     * @return BlockReceipt
+     */
+    public Request<?, BlockReceipts> getBlockReceipts(String blockHash) {
```

Comment:
This also use old `KlayTransactionReceipt.TransactionReceipt`

Reply:
I added BlockTransactionReceipts class